src/components/Header.js

```diff
@@ -61,7 +61,7 @@ const HeaderItems = headerContent.map(({ name, link }) => {
 
 const Header = () => {
   const classes = headerStyles();
-  const [open, setModalStage] = useState(false);
+  const [open, setModalStage] = useState(true);
```

Comment:
Would suggest a different pair of names. The convention is [a, setA] 

in this case, would it be modalStageOpen?